api/httpapi/handlers/asset_handler.go

```diff
@@ -61,9 +69,8 @@ func (h *AssetHandler) GetAll(w http.ResponseWriter, r *http.Request) {
 	withTotal, ok := r.URL.Query()["with_total"]
 	if ok && len(withTotal) > 0 && withTotal[0] != "false" && withTotal[0] != "0" {
 		total, err := h.assetRepo.GetCount(r.Context(), asset.Config{
```

Comment:
`GetCount` needs to have the same config value as `GetAll`, so we won't get the wrong total.